antarest/study/storage/variantstudy/model/command/remove_area.py

```diff
@@ -65,19 +65,15 @@ def _remove_area_from_sets_in_config(self, study_data_config: FileStudyTreeConfi
                     set_.areas.remove(self.id)
                     study_data_config.sets[id_] = set_
 
-    @override
-    def _apply_config(self, study_data_config: FileStudyTreeConfig) -> Tuple[CommandOutput, Dict[str, Any]]:
+    def remove_from_config(self, study_data_config: FileStudyTreeConfig) -> CommandOutput:
         del study_data_config.areas[self.id]
 
         self._remove_area_from_links_in_config(study_data_config)
         self._remove_area_from_sets_in_config(study_data_config)
 
         remove_area_cluster_from_binding_constraints(study_data_config, self.id)
 
-        return (
-            CommandOutput(status=True, message=f"Area '{self.id}' deleted"),
-            {},
-        )
+        return CommandOutput(status=True, message=f"Area '{self.id}' deleted")
```

Comment:
In this kind of case where there is no error case, I think it would be more straight forward to not return anything here, and create the `CommandOuput` at the end of the `_apply` method